packages/react-core/src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/Sidebar/sidebar';
+
+export interface SidebarProps extends React.HTMLProps<HTMLDivElement> {
+  children: React.ReactNode;
+  variant: 'default' | 'stack' | 'split' | 'panelRight';
+  hasGutter: boolean;
+  hasNoBackground: boolean;
+}
+
+export const Sidebar: React.FunctionComponent<SidebarProps> = ({
+  children,
+  variant = 'default',
+  hasGutter,
+  hasNoBackground,
+  ...props
+}: SidebarProps) => (
+  <div
+    className={css(
+      styles.sidebar,
+      hasGutter && styles.modifiers.gutter,
+      hasNoBackground && styles.modifiers.noBackground,
+      variant !== 'default' && styles.modifiers[variant]
```

Comment:
We don't support this now, but I plan on supporting responsive stack, split, gutter, and panel-right/panel-left variants in the future. So like `stack-on-md, split-on-lg`, etc.... `panel-left-on-md, panel-right-on-lg`, etc... `gutter-on-md, no-gutter-on-lg`, etc... would that change how this is built, or should we deal with that when we get there? `panel-right` doesn't seem like a variant with `stack` and `split`, either. I would expect to be able to do `split, stack-on-md, split-on-lg` and also `panel-left-on-lg`

Reply:
Split into orientation and `isPanelRight` props.